src/device-state/device-config.ts

```diff
@@ -396,6 +402,7 @@ function getConfigSteps(
 	target: Dictionary<string>,
 ) {
 	const configChanges: Dictionary<string> = {};
+	const rebootingChanges: Dictionary<string> = {};
```

Comment:
I'm a bit confused by the reboot changes in the device-config module. If a reboot breadcrumb step is being added where there was none before, how were device-config changes resulting in reboot before?